common/reactive/src/test/java/io/helidon/common/reactive/SingleTest.java

```diff
@@ -88,7 +88,7 @@ public void onSubscribe(Subscription subscription) {
         };
         Single.<String>just("foo").subscribe(subscriber);
         assertThat(subscriber.isComplete(), is(equalTo(false)));
-        assertThat(subscriber.getLastError(), is(nullValue()));
+        assertThat(subscriber.getLastError(), instanceOf(IllegalArgumentException.class));
```

Comment:
Although not verified by the TCK, ignoring non-positive request amounts after cancellation can hide subtle bugs in operators.

Reply:
Nice thx!